cmd/ketch/app_info.go

```diff
@@ -121,3 +128,32 @@ func appInfo(ctx context.Context, cfg config, options appInfoOptions, out io.Wri
 	fmt.Fprintf(out, "%v", buf.String())
 	return nil
 }
+
+func filterProcessDeploymentPods(appPods []corev1.Pod, deployment, process string) []corev1.Pod {
+	var pods []corev1.Pod
+Loop:
```

Comment:
`Loop:` is a kind of BFG in golang, If you know what I mean 

how about a more straightforward approach: 
```golang
func filterProcessDeploymentPods(appPods []corev1.Pod, deployment, process string) []corev1.Pod {
	var pods []corev1.Pod
	for _, pod := range appPods {
		version := pod.Labels[ketchDeploymentVersionLabel]
		processName := pod.Labels[ketchProcessNameLabel]
		if version == deployment && processName == process {
			pods = append(pods, pod)
		}
	}
	return pods
}

```
or we can group all pods by deployment and process before using them
```golang

// Pods of the application grouped by a deployment version and then by a process name
pods := make(map[string]map[string][]corev1.Pod)

for _, pod := range appPods.Items {
	version := pod.Labels[ketchDeploymentVersionLabel]
	byVersion, ok := pods[version]
	if !ok {
		byVersion = map[string][]corev1.Pod{}
		pods[version] = byVersion
	}
	processName := pod.Labels[ketchProcessNameLabel]
	byVersion[processName] = append(byVersion[processName], pod)
}


```